plugins/woocommerce/tests/e2e-pw/tests/admin-tasks/payment.spec.js

```diff
@@ -73,7 +73,7 @@ test.describe( 'Payment setup task', () => {
 		await page
 			.locator( '//input[@placeholder="BIC / Swift"]' )
 			.fill( 'ABBA' );
-		await page.locator( 'text=Save' ).click();
+		await page.getByRole( 'button', { name: 'Save' } ).click();
```

Comment:
`page.locator( 'text=Save' ).click();` was resolved to two elements on WPCOM. Make it more unique